channeldb/revocation_log.go

```diff
@@ -336,7 +373,11 @@ func serializeHTLCEntries(w io.Writer, htlcs []*HTLCEntry) error {
 
 // deserializeRevocationLog deserializes a RevocationLog based on tlv format.
 func deserializeRevocationLog(r io.Reader) (RevocationLog, error) {
-	var rl RevocationLog
+	var (
```

Comment:
the serialization and deserialization are missing unit tests to show the compatibility re the added fields.

Reply:
The current unit tests cover the new fields given that I have adapted `testChannelCommit` and `testRevocationLogBytes`.